public/resources/core/js/services/party.individual.service.js

```diff
@@ -83,19 +94,13 @@
             var result = '';
 
             switch (this.type) {
-            case TYPES.CONTACT_MEDIUM.EMAIL_ADDRESS:
+            case TYPES.CONTACT_MEDIUM.EMAIL_ADDRESS.code:
                 result = this.medium.emailAddress;
                 break;
-            case TYPES.CONTACT_MEDIUM.POSTAL_ADDRESS:
-                result = [
-                    this.medium.streetOne,
-                    this.medium.stateOrProvince,
-                    this.medium.postcode,
-                    this.medium.city,
-                    this.medium.country
-                ].join(', ');
+            case TYPES.CONTACT_MEDIUM.POSTAL_ADDRESS.code:
+                result = this.medium.streetOne + ', ' + this.medium.city + '\n' + this.medium.postcode + ', ' + this.medium.stateOrProvince + '\n' + parseCountry(this.medium.country);
```

Comment:
result = this.medium.streetOne + '\n' + this.medium.postcode + ' ' + this.medium.city + ' (' + this.medium.stateOrProvince + ') \n' + parseCountry(this.medium.country);